api/src/main/java/uk/gov/hmcts/payment/api/controllers/CreditAccountPaymentController.java

```diff
@@ -210,7 +215,14 @@ public String return504(AccountServiceUnavailableException ex) {
     }
 
     private boolean isAccountStatusCheckRequired(Service service) {
-        return ff4j.check("credit-account-payment-liberata-check") && Service.FINREM == service;
+        if (ff4j.check("credit-account-payment-liberata-check")) {
+            if (ff4j.check("check-liberata-account-for-all-services")) {
```

Comment:
what if only check-liberata-account-for-all-services is enabled?

Reply:
the test givenLiberataCheckOffAndCheckLiberataAccountForAllSericesOffThenNoServiceTriggersLiberataCheck() covers that case